build/webpack/webpack.client.config.js

```diff
@@ -18,21 +18,25 @@ module.exports = {
     output: {
         path: path.join(constants.ExtensionRootDir, 'out', 'client_renderer'),
         filename: '[name].js',
-        chunkFilename: `[name].bundle.js`
+        chunkFilename: `[name].bundle.js`,
+        libraryTarget: 'module'
     },
+    experiments: {
+        outputModule: true
+    },
+    target: 'node',
```

Comment:
Webpack 5 doesn't shim build in node modules. Per the warning message here it seems like we could either reference the shim package and add a fallback or if we target 'node' I don't believe that it's needed. Was a tiny bit unsure on this, but this seemed to work with the node loaded, so went with this for now versus referencing shim packages.

Here is an example of the type of error we would see after the 4-5 update.